<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNewTermDialog</class>
 <widget class="QDialog" name="AddNewTermDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add new term</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelTerm">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Term&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditInputTerm"/>
        </item>
        <item>
         <widget class="QLabel" name="labelDefinition">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Definiton&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditInputDefinition"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonAddNewTerm">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
            <underline>false</underline>
           </font>
          </property>
          <property name="text">
           <string>Add</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
